module.exports = function syncException () {
  throw new Error('Error thrown')
}
